crates/proc-macro-srv/src/abis/abi_1_64/mod.rs

```diff
@@ -0,0 +1,104 @@
+//! Macro ABI for version 1.63 of rustc
```

Comment:
shouldn't that read 1.64?

Reply:
Yeah, removed it in https://github.com/rust-lang/rust-analyzer/pull/12802